src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,19 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
     public static void main(String[] args) {
+        final Logger logger = LogManager.getLogger(Main.class);
+
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("AuthenticationService failed, Reason: {}" + e.getMessage());
```

Comment:
The logger error message should use placeholders instead of string concatenation. Change `logger.error("AuthenticationService failed, Reason: {}" + e.getMessage());` to `logger.error("AuthenticationService failed, Reason: {}", e.getMessage());` to follow best practices for logging.